src/tutorials/volume-snapshots-how-to-create-and-use-data-clones-in-okteto.mdx

```diff
@@ -0,0 +1,311 @@
+description: Learn how to create a data clone of an application’s database, and how to use it when creating a new Development Environment.
+---
+
+import Image from '@theme/Image';
+
+Data cloning allows you to create an independent, identical copy of an application’s database, making it easy to test with realistic data.
+Using Okteto Volume Snapshots, you can quickly create development environments that include production or staging data—without manually copying databases or waiting for CI/CD pipelines.
+
+:::tip
+Volume Snapshots are also useful for loading test data when running tests in Okteto Test. If your tests require a database with preloaded data, you can configure your test environment to restore from a snapshot instead of initializing an empty database. This ensures consistency across test runs and reduces setup time.
+:::
+
+In this tutorial, you'll:
+✅ deploy an application to Okteto
+✅ learn how to create a data clone of the application's database
+✅ and then learn how to use it when creating a new Development Environment
+
+## Prerequisites
+
+In order to follow this tutorial, you should have:
+
+-   A **Kubernetes cluster with Okteto** installed
+-   Familiarity with **Kubernetes manifests**
+-   [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine
+
+## Step 1: Deploy the Sample Application
+
+For this tutorial, we’ll deploy a CRUD recipe application that stores data in MongoDB.
+
+Clone the repository:
+```console
+$ git clone https://github.com/okteto/fastapi-snapshot-app
+$ cd fastapi-snapshot-app
+```
+
+In the `manifests` folder, the `k8s.yml` manifest file is responsible for the deployment of the application's service, and the `mongo.yml` manifest file is responsible for deploying the database service. The database service in this case is a MongoDB service.
+
+Apply the manifests to deploy the application:
+
+```console
+okteto kubeconfig
+kubectl apply -f manifests/
+```
+
+Expected output:
+```console
+deployment.apps/fastapi-snapshot-demo created
+service/fastapi-snapshot-demo created
+persistentvolumeclaim/data created
+service/mongodb created
+statefulset.apps/mongodb created
+```
+
+<Image
+  src={require("@site/static/img/tutorials/volume-snapshots/deployed.png").default}
+  alt="Deployed Application"
+  width="1000"
+/>
+
+✅ Your application is now running in Okteto. Take note of the **endpoint URL** under **Endpoints** in the Okteto Dashboard.
+
+
+## Step 2: Populate the Application Database
+
+Before taking a snapshot, add sample data. This is the data that we'll be including in the Development Environments that we'll create in the next steps.
+
+```console
+curl -X 'POST' \
+  'https://fastapi-snapshot-demo-yournamespace.YOURSUBDOMAIN/recipe' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "name": "Donuts",
+  "ingredients": ["Flour", "Milk", "Sugar", "Vegetable Oil"]
+}'
+```
+
```

Comment:
nit: how about a first step to put the endpoint on an env var, and then we use that for all the curls?